### PR TITLE
Doctor jekyll reports conflicts

### LINKED ISSUE #7844: "Get customer authorization token" page does have a content
# Bug report

## Description

[Get customer authorization token](https://devdocs.magento.com/guides/v2.3/graphql/get-customer-authorization-token.html) page does have a content.

@keharper , could you please confirm the issue?

Because I see you removed all content from this page in https://github.com/magento/devdocs/commit/4524736df367b26579af02f9c5161b4ea309ab4a#diff-0b6fb511fd9b8b007e7802627358b4adL2.

Thank you!

### Steps to reproduce

1. Open
- https://devdocs.magento.com/guides/v2.3/graphql/get-customer-authorization-token.html
- https://devdocs.magento.com/guides/v2.4/graphql/get-customer-authorization-token.html

2. Page exists but contains an empty content.

## Expected result

The expected page content is: https://raw.githubusercontent.com/magento/devdocs/502840ce4e1c8a6b2121aef5f2ee9c9da3438239/guides/v2.3/graphql/get-customer-authorization-token.md

### PR DESCRIPTION
**Pull request by BarnyShergold**
## Purpose of this pull request

Fix errors (where possible) caused when running : `bin/jekyll doctor`

## Affected DevDocs pages
Deleted as file no longer required : /guides/v2.4/install-gde/continue-to-verify.md
Deleted as file no longer required : /guides/v2.4/install-gde/install-roadmap_web.md
Deleted as file no longer required : /guides/v2.3/graphql/get-customer-authorization-token.md
Deleted as file no longer required : /guides/v2.3/graphql/get-customer-authorization-token.md

FIXED - NO LONGER SYMLINK /guides/v2.3/graphql/get-customer-authorization-token.html

SYMLINK /guides/v2.3/ui-components/ui-listing-grid.html
SYMLINK /guides/v2.3/config-guide/config/caching_frontend-cache-types.html
SYMLINK /guides/v2.3/config-guide/deployment/pipeline/example/rsync.html
SYMLINK /guides/v2.3/graphql/reference/cms.html
FIXED - NO LONGER SYMLINK /graphql/develop/configure-graphql-xml.html
SYMLINK /guides/v2.3/graphql/reference/custom-attribute-metadata.html
SYMLINK /guides/v2.3/graphql/reference/sales.html
SYMLINK /guides/v2.3/graphql/reference/vault.html
SYMLINK /guides/v2.3/config-guide/database/database.html
SYMLINK /guides/v2.3/config-guide/cache/caching-database.html
SYMLINK /guides/v2.3/extension-dev-guide/backward-compatibility.html
SYMLINK /guides/v2.3/graphql/reference/directory.html

cloud/docker/docker-manage-cron-jobs.md (wrong files) /cloud/reference/docker-config.html
cloud/cdn/fastly-vcl-bypass-to-origin.md (wrong file) /cloud/configure/fastly-vcl-badreferer.html

SYMLINK /guides/v2.3/graphql/search-pagination.html
SYMLINK /guides/v2.3/graphql/queries.html
SYMLINK /guides/v2.3/architecture/versioning.html
SYMLINK /guides/v2.3/graphql/mutations.html
SYMLINK /guides/v2.3/config-guide/prod/prod_perf-optimize.html
SYMLINK /guides/v2.3/config-guide/prod/performance-best-practices.html
SYMLINK /guides/v2.3/performance-best-practices.html
SYMLINK /guides/v2.3/inventory/catalog-inventory-replacements.html
SYMLINK /guides/v2.3/inventory/scalable-inventory-replacements.html
SYMLINK /guides/v2.3/install-gde/trouble/tshoot_mysql_table-open-cache.html
SYMLINK /guides/v2.3/install-gde/trouble/php/tshoot_phpini.html
SYMLINK /guides/v2.3/get-started/order-tutorial/order-create-customer.html
SYMLINK /guides/v2.3/get-started/order-tutorial/order-create-invoice.html
SYMLINK /guides/v2.3/get-started/order-tutorial/order-create-order.html
SYMLINK /guides/v2.3/get-started/order-tutorial/order-create-quote.html
SYMLINK /guides/v2.3/get-started/order-tutorial/order-create-shipment.html
SYMLINK /guides/v2.3/get-started/order-tutorial/order-issue-refund.html
SYMLINK /guides/v2.3/ui-components/ui-form.html
SYMLINK /guides/v2.3/graphql/reference/url-resolver.html
SYMLINK /guides/v2.3/graphql/reference/wishlist.html

As reported on #7164 - All left are due to 2.3 documents symlinking to 2.4

Fixes #7844

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by BarnyShergold** [comment]
Took quite a while to get through all these files!

File: src/guides/v2.4/install-gde/prereq/mysql.md
Comment (meker12) [inline src/guides/v2.4/install-gde/prereq/mysql.md:5]: You can remove this redirect from the v2.4 file. It doesn't work.

Works fine for v2.3
https://devdocs.magento.com/guides/v2.3/install-gde/trouble/tshoot_mysql_table-open-cache.html -- redirects to https://devdocs.magento.com/guides/v2.3/install-gde/prereq/mysql.html

https://devdocs.magento.com/guides/v2.4/install-gde/prereq/mysql.html -- redirects to Page not found

File: src/guides/v2.3/install-gde/prereq/php-settings.md
Comment (meker12) [inline src/guides/v2.3/install-gde/prereq/php-settings.md:9]: Not sure why you are removing the `/guides/v2.3`. -- this redirect seems to be working fine.

Maybe you meant to remove the redirect section from the  file `src/guides/v2.4/install-gde/prereq/php-settings.md`

**Review by meker12 — CHANGES_REQUESTED** [review]

**Comment by meker12** [comment]
Added @keharper to review the changes to the GraphQL files

**Comment by dshevtsov** [comment]
Adding @jeff-matthews since installation guide was affected.

File: src/guides/v2.4/install-gde/prereq/mysql.md
Comment (jeff-matthews) [reply src/guides/v2.4/install-gde/prereq/mysql.md:5]: @meker12, I think you mean that `http://devdocs.magento.com/guides/v2.4/install-gde/trouble/tshoot_mysql_table-open-cache.html` returns a 404.

File: src/cloud/cdn/fastly-vcl-bypass-to-origin.md
Comment (BarnyShergold) [reply src/cloud/cdn/fastly-vcl-bypass-to-origin.md:5]: Redirect removed from this file

File: src/cloud/docker/docker-manage-cron-jobs.md
Comment (BarnyShergold) [reply src/cloud/docker/docker-manage-cron-jobs.md:8]: Redirect removed from this file

File: src/guides/v2.4/install-gde/prereq/mysql.md
Comment (BarnyShergold) [reply src/guides/v2.4/install-gde/prereq/mysql.md:5]: @jeff-matthews @meker12 
Yes - as this file is not a Symlinked file anymore, it needed to have 2.4 added. 
I've changed to guides/v2.4/install-gde/trouble/tshoot_mysql_table-open-cache.html to match the format in 2.3 version of file

File: src/guides/v2.3/install-gde/prereq/php-settings.md
Comment (BarnyShergold) [reply src/guides/v2.3/install-gde/prereq/php-settings.md:9]: Yes 2.4 needed changing

File: src/guides/v2.3/graphql/queries/products.md
Comment (keharper) [inline src/guides/v2.3/graphql/queries/products.md:5]: I don't understand why you've made the path shorter here, but longer in `create-graphqls-file.md`. The original looks correct.

File: src/guides/v2.3/install-gde/prereq/php-settings.md
Comment (keharper) [inline src/guides/v2.3/install-gde/prereq/php-settings.md:10]: Remove empty line

File: src/guides/v2.4/graphql/authorization-tokens.md
Comment (keharper) [inline src/guides/v2.4/graphql/authorization-tokens.md:7]: Remove the redirect. A 2.4 version of `get-customer-authorization-token.md` has never existed.

File: src/guides/v2.4/install-gde/prereq/php-settings.md
Comment (keharper) [inline src/guides/v2.4/install-gde/prereq/php-settings.md:9]: Remove this redirect. The file never existed in 2.4.

File: src/guides/v2.4/install-gde/prereq/mysql.md
Comment (keharper) [inline src/guides/v2.4/install-gde/prereq/mysql.md:5]: Remove the redirect. The file has never existed in 2.4.

**Review by keharper — CHANGES_REQUESTED** [review]
I can't speak for the cloud docs, but I know the GraphQL docs and did the research on the installation docs.

File: src/guides/v2.3/install-gde/prereq/php-settings.md
Comment (BarnyShergold) [reply src/guides/v2.3/install-gde/prereq/php-settings.md:10]: Done

File: src/guides/v2.4/graphql/authorization-tokens.md
Comment (BarnyShergold) [reply src/guides/v2.4/graphql/authorization-tokens.md:7]: It was symlinked - so the URL has existed. This is borne out by the fact if you visit  https://devdocs.magento.com/guides/v2.4/graphql/get-customer-authorization-token.html you get an empty page not a 404

Plus the new file - authorization-tokens.md has different 2.3 & 2.4 files. This file has also been complained about on issue #7844 
For safety I think this should stand.

File: src/guides/v2.3/graphql/queries/products.md
Comment (BarnyShergold) [reply src/guides/v2.3/graphql/queries/products.md:5]: Fixed

File: src/guides/v2.4/install-gde/prereq/php-settings.md
Comment (BarnyShergold) [reply src/guides/v2.4/install-gde/prereq/php-settings.md:9]: Fixed

File: src/guides/v2.4/install-gde/prereq/mysql.md
Comment (BarnyShergold) [reply src/guides/v2.4/install-gde/prereq/mysql.md:5]: Fixed

File: src/guides/v2.3/graphql/queries/products.md
Comment (dshevtsov) [inline src/guides/v2.3/graphql/queries/products.md:5]:
```suggestion
  - /guides/v2.3/graphql/reference/products.html
```

File: src/guides/v2.3/graphql/develop/create-graphqls-file.md
Comment (dshevtsov) [inline src/guides/v2.3/graphql/develop/create-graphqls-file.md:4]:
```suggestion
redirect_from: /guides/v2.3/graphql/develop/configure-graphql-xml.html
```

File: src/guides/v2.4/graphql/develop/create-graphqls-file.md
Comment (dshevtsov) [inline src/guides/v2.4/graphql/develop/create-graphqls-file.md:4]:
```suggestion
redirect_from: /guides/v2.4/graphql/develop/configure-graphql-xml.html
```

**Review by dshevtsov — CHANGES_REQUESTED** [review]
Minor suggestions to comply with examples at https://github.com/jekyll/jekyll-redirect-from

**Review by dshevtsov — APPROVED** [review]

**Comment by BarnyShergold** [comment]
@dshevtsov - Now that you have approved this, do I have to wait until keharper & meker12 re-review or am I allowed to now dismiss their reviews?

**Comment by dshevtsov** [comment]
@BarnyShergold, let's wait for approvals from @keharper and @meker12

**Comment by dshevtsov** [comment]
@BarnyShergold, feel free to open a new PR to add content such as aeb41e1, since it's out of scope here considering the PR's purpose.

File: src/guides/v2.4/graphql/authorization-tokens.md
Comment (meker12) [reply src/guides/v2.4/graphql/authorization-tokens.md:7]: @keharper  Can you check this comment?  I think we can get this approved if we address this outstanding comment.

**Review by meker12 — APPROVED** [review]

**Comment by BarnyShergold** [comment]
> GraphiQL ==> GraphQL?

Corrected - this is copied content so is wrong elsewhere

**Review by meker12 — APPROVED** [review]

**Review by keharper — APPROVED** [review]

**Comment by meker12** [comment]
running tests

**Comment by keharper** [comment]
"GraphiQL" was not wrong, because it was a screenshot from the GraphiQL browser. It's OK to leave it as GraphQL because the browser name isn't all that important.

**Pull request merged**
